include/config/config_widescreen.h

```diff
@@ -0,0 +1,37 @@
+#ifndef CONFIG_WIDESCREEN_H
+#define CONFIG_WIDESCREEN_H
+
+// This file provides macros to handle switching between 4:3 and 16:9 and fixing issues.
+// It is not meant to be something the users can edit.
```

Comment:
i'd suggest putting this file just under `/include/widescreen.h` instead in order to not confuse people that this is a config file